README.md

```diff
@@ -239,10 +239,17 @@ byte[] secretA = encapsulationResult.getSecret();
 EncryptedMessage encryptedMessage = encapsulationResult.getEncryptedMessage();
 ```
 
-The encrypted message `encryptedMessage` is transported back to `BOB` who uses the public key and cipher text included in the message for the decapsulation phase of KEM:
+The encrypted message `encryptedMessage` is converted into a byte array representation `encodedMessage` and transported back to `BOB` 
+who uses the public key and cipher text included in the message for the decapsulation phase of KEM.
```

Comment:
Turn on "Soft word-wrap" in the editor and use one long line. This hard line-wrap makes it difficult to edit and make changes.